source/index.rst

```diff
@@ -21,6 +21,8 @@ Basic GIS operations
 * :doc:`docs/3/advanced_georeferencing`
 * :doc:`docs/3/digitizing_basics`
 * :doc:`docs/3/downloading_osm_data`
+* :doc:`docs/3/basic_filtering_and_styling_with_expressions`
```

Comment:
Start a new section after Advanced GIS Operations called 'Expressions'. This tutorial will be part of that new section. The new tutorial need s to be added to toctree section at the bottom also.